content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/start.rb

```diff
@@ -17,6 +17,7 @@ def main
                 @handle.log('info', msg)
                 @handle.create_notification(:level => 'info', :subject => service, :message => msg)
                 @handle.root['ae_result'] = 'ok'
+                check_interval
```

Comment:
I would not call this method `check_interval` since it is setting, not checking and the name does not clarify what type of interval.  A better name would be `calculate_retry_internal` and reduce the method to just returning the integer value.

This line would become:
`@handle.root['ae_retry_interval'] = calculate_retry_internal`